docusaurus/docs/get-started/get-started.mdx

```diff
@@ -117,7 +117,7 @@ Run the following command and answer the prompts:
   queryString="current-package-manager"
 />
 
-For help with the prompts, refer to the [Prompts reference](../reference/prompts.md).
+For help with the prompts, refer to the [Prompts reference](../reference/cli-commands.mdx).
```

Comment:
we are renaming the file to "cli commands" but keeping here "prompts reference" is that intended?

Reply:
Hmm, yeah, it was intended, but I am not sure it's correct, I was a bit in two minds about this one. What do you think?

Reply:
Just confirming. The page has the correct information after all.

Reply:
I'd update the link text personally

Reply:
Sure, changed in https://github.com/grafana/plugin-tools/pull/1042/commits/b93da3edc1068ddcf7147281506c87e5306c72ba.